packages/core/botpress/src/web/util/Toast.js

```diff
@@ -0,0 +1,28 @@
+import EventBus from '~/util/EventBus'
+import { toast } from 'react-toastify'
+
+export default class Toast {
+  constructor({ title }) {
```

Comment:
The name "title" suggests a human-readable thing, should it instead be "id" or "code"?